### DISCUSSION TITLE
Animated 2D Textures

### DISCUSSION THREAD

**Discussion by sprengerst** [google/model-viewer#3136] [Q&A]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

Hi,

would it be possible to include 2D animated texture options? 

If yes, how would that perform in WebXR and Quicklook. I think "auto generate usdz" would be a problem to preserve such an gif animation?

Thanks,
Stefan

#### Live Demo

https://glitch.com/edit/#!/model-viewer

https://github.com/MaciejWWojcik/three-plain-animator

### Version

- model-viewer: v1.10.1

### Browser Affected

- [X ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ X] Windows

### AR

- [X ] WebXR
- [X ] SceneViewer
- [ X] QuickLook

**Comment by samaneh-kazemi** [comment]
@elalish any input?

**Reply by elalish** [reply]
@sprengerst No, glTF doesn't support animated textures yet. What is your use  case? I'd recommend getting involved with the  Khronos 3D Formats WG and  draft an extension for animated textures.  You'll get good feedback from the community about how to standardize it.
